staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/admission.go

```diff
@@ -224,6 +185,7 @@ func (a *GenericAdmissionWebhook) loadConfiguration(attr admission.Attributes) (
 	return hookConfig, nil
 }
 
+// TODO: move this object to a common package
```

Comment:
What is "common"? webhook/common? I.e. for validating and mutating. Yes please.

Reply:
Please see it in my followup PR today.